client.js

```diff
@@ -0,0 +1,143 @@
+const http = require('http');
+const net = require('net');
+let args = process.argv;
+
+/**********************
+    User Arguments
+**********************/
+
+if (!args[2]) {
+  console.log(`
```

Comment:
Putting a return here will turn this into a guard so that the rest of your code doesn't have to be in an else conditional.